<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HTM - Task Execution Using Language</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QWidget" name="virtualWorld" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="controlPanel" native="true">
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="grpBx_playerControls">
         <property name="title">
          <string>Controles del personaje</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QWidget" name="wgt_directionControls" native="true">
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="1">
              <widget class="QPushButton" name="btn_down">
               <property name="text">
                <string>Aba.</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="btn_up">
               <property name="text">
                <string>Arr.</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="btn_right">
               <property name="text">
                <string>Der.</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="btn_left">
               <property name="text">
                <string>Izq.</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="btn_grab">
               <property name="text">
                <string>Coger</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="ln_playerControls">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="wgt_otherControls" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="btn_dance">
               <property name="text">
                <string>Bailar</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="grpBox_worldControls">
         <property name="title">
          <string>Controles del mundo</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QComboBox" name="cmbBx_insertObj">
            <item>
             <property name="text">
              <string>Hat-Hunter</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hat-Nurse</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="btn_insertObj">
            <property name="text">
             <string>Insertar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tbWdgt_userControls">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tb_execute">
          <attribute name="title">
           <string>Ejecutar</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="1">
            <widget class="QPushButton" name="btn_execSentence">
             <property name="text">
              <string>Ejecutar</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lbl_log">
             <property name="text">
              <string>Log:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lbl_sentence_execute">
             <property name="text">
              <string>Frase:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lnEdt_sentence_execute"/>
           </item>
           <item row="2" column="1">
            <widget class="QTextEdit" name="txtEdt_log">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="btn_clearLog">
             <property name="text">
              <string>Limpiar</string>
             </property>
            </widget>
           </item>
          </layout>
          <zorder>lbl_sentence_execute</zorder>
          <zorder>lnEdt_sentence_execute</zorder>
          <zorder>btn_execSentence</zorder>
          <zorder>lbl_log</zorder>
          <zorder>txtEdt_log</zorder>
          <zorder>btn_clearLog</zorder>
         </widget>
         <widget class="QWidget" name="tb_train">
          <attribute name="title">
           <string>Entrenar</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QWidget" name="wdgt_trainInputs" native="true">
             <layout class="QGridLayout" name="gridLayout_2">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <property name="horizontalSpacing">
               <number>6</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="lbl_sentence_train">
                <property name="text">
                 <string>Frase:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lbl_action">
                <property name="text">
                 <string>Accion:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="cmbBx_action">
                <item>
                 <property name="text">
                  <string>Mover arriba</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mover abajo</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mover derecha</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mover izquierda</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Bailar</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Recoger</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lnEdt_sentence_train"/>
              </item>
              <item row="2" column="1">
               <widget class="QPushButton" name="btn_addTrainInput">
                <property name="text">
                 <string>Añadir</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tblWdgt_trainingList">
             <property name="columnCount">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <item>
               <widget class="QLabel" name="lbl_iterations">
                <property name="text">
                 <string>Iterations</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnEdt_iterations">
                <property name="layoutDirection">
                 <enum>Qt::LeftToRight</enum>
                </property>
                <property name="text">
                 <string>10</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_clear_train">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Limpiar Tabla</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="wdgt_trainButtons" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="leftMargin">
               <number>3</number>
              </property>
              <property name="topMargin">
               <number>3</number>
              </property>
              <property name="rightMargin">
               <number>3</number>
              </property>
              <property name="bottomMargin">
               <number>3</number>
              </property>
              <item>
               <widget class="QPushButton" name="btn_import">
                <property name="text">
                 <string>Importar...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_train">
                <property name="text">
                 <string>Entrenar</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>controlPanel</zorder>
   <zorder>virtualWorld</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
